README.md

```diff
@@ -2,7 +2,7 @@
 # My Portfolio
 
 ## Live demo link
-follow the link below to my portfolio live page
+##### follow the link below to my portfolio live page
 [Kalung'e.com](https://kalunge.github.io/portfolio-project/)
 
 ## Built With
```

Comment:
I think your portfolio could use a preview screenshot somewhere over here. Not every person that visits your repo will go to the Live Demo so having a preview screenshot of your work is a good idea to make them attracted to know more about you.